Makefile

```diff
@@ -89,6 +90,8 @@ deploy-no-build: deploy-setup
 deploy-example: deploy
 	oc create -n $(NAMESPACE) -f hack/cr.yaml
 
+test-unit: 
+	@go test $(PKGS)
```

Comment:
+100 on adding unit tests, can we add them to another package directory other than `k8shander` though?

Reply:
I grabbed the change from another operator so it should glob test files in any of the packages except those here: https://github.com/openshift/cluster-logging-operator/pull/103/files#diff-b67911656ef5d18c4ae36cb6741b7965R19

Reply:
right, im just asking that maybe we do something like `/pkg/unittests/` or something under `/pkg` so that it is easily found and distinguished from the e2e tests

Reply:
@ewolinetz this is not how 'unittests' are organized in golang.  They live alongside the code, most likely because of visibility requirements.